samples/SpectatorView.Example.Unity/README.md

```diff
@@ -1,3 +1,70 @@
+
+#### Configure your project
+
+1. Open the `SpectatorView.Example.Unity` project in Unity.
+2. If building for HoloLens 2, ensure that `QRCODESTRACKER_BINARY_AVAILABLE` has been defined as a Scripting Define Symbols in your WSA Unity Player Settings. If you have made no changes to the SpectatorView.Examples.Unity project definition, this should already be defined (Player settings can be found under `Build Settings -> Player Settings`).
+3. If building for HoloLens 1, ensure that `SpectatorView.OpenCV.dll` exists in the project's Assets folder. It should have been placed under Assets\MixedReality-SpectatorView\SpectatorView.Native\Plugins\WSA\x86 when going through the compilation instructions [here](../../src/SpectatorView.Native/README.md).
+
+#### Build & Deploy
+
+##### HoloLens
+
+1. Select `HoloLens` as your target platform using Spectator View's `PlatformSwitcher` (This can be found in the Unity inspector for the `SpectatorView` prefab GameObject).
+2. Build UWP Player containing `MixedReality-SpectatorView/SpectatorView/Scenes/SpectatorView.HoloLens` scene, and deploy this application to the HoloLens device.
+3. Launch `SpectatorView.Example.Unity` on the HoloLens and wait for the experience to start.
+
+![Marker](./../../doc/images/HoloLensSpectatorViewExampleBuildSettings.png)
+
+##### Android
+
+1. Select `Android` as your target platform using Spectator View's `PlatformSwitcher` (This can be found in the Unity inspector for the `SpectatorView` prefab GameObject).
+2. Include and declare `MixedReality-SpectatorView/SpectatorView/Scenes/SpectatorView.Android.unity` as Scene 0 in your Unity Build Settings.
+3. Include and declare `MixedReality-SpectatorView/SpectatorView/Scenes/SpectatorView.HoloLens.unity` as Scene 1 in your Unity Build Settings.
+4. `Build and Run` the Android Player through Unity's Build Settings.
+5. Launch `SpectatorView.Example.Unity` on the Android device and connect to the HoloLens device by specifying the HoloLens's IP Address.
+
+![Marker](./../../doc/images/AndroidSpectatorViewExampleBuildSettings.png)
+
+##### iOS
+
+1. Select `iOS` as your target platform using Spectator View's `PlatformSwitcher` (This can be found in the Unity inspector for the `SpectatorView` prefab GameObject).
+2. Include and declare `MixedReality-SpectatorView/SpectatorView/Scenes/SpectatorView.iOS.unity` as Scene 0 in your Unity Build Settings.
+3. Include and declare `MixedReality-SpectatorView/SpectatorView/Scenes/SpectatorView.HoloLens.unity` as Scene 1 in your Unity Build Settings.
+4. `Build` the iOS Player through Unity's Build Settings.
+5. Launch `SpectatorView.Example.Unity` on the iOS device and connect to the HoloLens device by specifying the HoloLens's IP Address.
+
+![Marker](./../../doc/images/iOSSpectatorViewExampleBuildSettings.png)
+
+> Note 1: Additional setup instructions and troubleshooting for spectating with mobile devices can be found [here](../../doc/SpectatorView.Setup.md).
+
+## Sample Project Contents
```

Comment:
This might be helpful to have at the top, instead of much later (i.e. show a person what they are going to build and expect ahead of time to set expectations and give motivation for what they are building).

In general I feel like other good tutorials I see will set some expectation for the end result (i.e. what we're building) and then build up toward that.